subprojects/docs/src/snippets/ivy-publish/distribution/kotlin/build.gradle.kts

```
@@ -28,7 +28,7 @@ distributions {
 publishing {
     publications {
         create<IvyPublication>("myDistribution") {
-            artifact(tasks.distZip.get())
+            artifact(tasks.distZip)
```

TaskProvider is only supported by MavenPublication
```suggestion
            artifact(tasks.distZip.get())
```